tests/dotnet/UnitTests/PostBuildTest.cs

```diff
@@ -99,6 +99,26 @@ public void AssemblyStripping (string project, ApplePlatform platform, string ru
 			Assert.That (Path.Combine (appPath, "Microsoft.iOS.dll"), Does.Exist, "Platform Assembly");
 		}
 
+		[Test]
+		[TestCase ("MySimpleApp", ApplePlatform.MacCatalyst, "maccatalyst-arm64;maccatalyst-x64", true)]
+		[TestCase ("MySimpleApp", ApplePlatform.MacCatalyst, "maccatalyst-arm64;maccatalyst-x64", false)]
+		public void DefaultAssemblyStripping (string project, ApplePlatform platform, string runtimeIdentifiers, bool shouldStrip)
+		{
+			Configuration.IgnoreIfIgnoredPlatform (platform);
+			Configuration.AssertRuntimeIdentifiersAvailable (platform, runtimeIdentifiers);
+
+			var project_path = GetProjectPath (project, runtimeIdentifiers: runtimeIdentifiers, platform: platform, out var appPath);
+			Clean (project_path);
+			var properties = GetDefaultProperties (runtimeIdentifiers);
+
+			// Verify value defaults to false when not set
+			properties ["EnableAssemblyILStripping"] = shouldStrip ? "true" : "";
```

Comment:
This doesn't actually test the default value, since you're setting the value in the test.

You need to build for "Release", _not_ set `EnableAssemblyILStripping`, and then verify that the app's assemblies are not stripped (when building for two runtime identifiers).